pkg/apis/provisioning/v0alpha1/types.go

```diff
@@ -28,30 +28,25 @@ type LocalRepositoryConfig struct {
 type Workflow string
 
 const (
-	// BranchWorkflow creates a branch for changes
+	// WriteWorkflow allows users to write directly to the respository
+	WriteWorkflow Workflow = "write"
```

Comment:
I like `write` more than `push`. It's also more generic term for other types of repositories.